backend/tests/load_testing/config.py

```
@@ -0,0 +1,273 @@
+@dataclass
+class LoadTestConfig:
+    """Configuration for load testing."""
+    
+    # Target environment
+    base_url: str = ""
+    environment: str = "staging"
+    
+    # Authentication
+    test_user_prefix: str = "loadtest_user_"
+    test_user_domain: str = "@test.com"
+    test_password: str = "testpassword123"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "test_password"

(S105)